% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
  12pt,
]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
  \setmainfont[]{Arial}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage[left]{lineno}
\linenumbers
\usepackage{setspace}
\doublespacing
\usepackage{float}
\floatplacement{figure}{H}
\usepackage{indentfirst}
\usepackage{fixltx2e}
% https://github.com/rstudio/rmarkdown/issues/337
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

% https://github.com/rstudio/rmarkdown/pull/252
\usepackage{titling}
\setlength{\droptitle}{-2em}

\pretitle{\vspace{\droptitle}\centering\huge}
\posttitle{\par}

\preauthor{\centering\large\emph}
\postauthor{\par}

\predate{\centering\large\emph}
\postdate{\par}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newenvironment{cslreferences}%
  {}%
  {\par}

\date{}

\begin{document}

\textbf{Article title}: Seed plant families with diverse mycorrhizal
states have higher diversification rates

\textbf{Short running title}: Higher diversification in mycorrhizal
diverse plant families

\textbf{Keywords}: diversification rates, mycorrhizal states, seed
plants, key innovation, mycorrhizal diversity

\textbf{Type of article}: Article

\textbf{Number of words}: Abstract (200), Main Text (3727)

\textbf{Number of references}: 43

\textbf{Number of figures}: 4

\textbf{Number of tables}: 0

\pagebreak

\hypertarget{abstract}{%
\section{Abstract}\label{abstract}}

One crucial innovation in plant evolution was the association with soil
fungi during land colonization. Today, this symbiotic interaction is
present in most of plants species and can be classified in four types:
Arbuscular (AM), Ecto (EM), Orchid (OM) and Ericoid Mycorrhiza (ER).
Since the AM ancestral state, some plants lineages have switched partner
(EM, OM and ER) or lost the association (no-association: NM).
Evolutionary transitions to a novel mycorrhizal state (MS) might allow
plant lineages to access new resources, enhancing diversification rates.
However, some clades are not restricted to one MS, and this variability
might promote diversification. In this study we address the relationship
between MS and diversification rates of seed plant families. For this,
we used the recently published FungalRoot database, which complied data
for 14,870 species and their mycorrhizal partners. We assigned a single
MS to each plant family, calculated the MS heterogeneity and estimated
their diversification rates using the method-of-moments. Families with
mixed MS had the highest diversification rates and there was a positive
relationship between MS heterogeneity and diversification rates. These
results support the hypothesis that MS lability promotes diversification
and highlight the importance of the association with soil fungi for the
diversification of plants.

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

Understanding the basis of the exceptional plant diversity has been a
matter of interest for ecologist and evolutionary biologist since
Darwin. Great focus has been placed on estimating plants diversification
rates and identifying the factors that could influence them (Eriksson
and Bremer 1992; Moore and Donoghue 2007; O'Meara et al. 2016; Vamosi et
al. 2018). The acquisition of novel traits (sometimes referred to as
``key innovations''), such as pollination by animals (Eriksson and
Bremer 1992) or physiological seed dormancy (Willis et al. 2014), have
been proposed to promote diversification of plant lineages. This ``key
innovation'' perspective suggests that the acquisition of a novel trait
might allow a given lineage to exploit the environment in a
significantly different way, potentially resulting in an explosive
radiation.

One crucial innovation in plant evolution was the association with soil
fungi during land colonization (Pirozynski and Malloch 1975; Selosse and
Le Tacon 1998; Strullu-Derrien et al. 2018). Before plant colonization,
land was hostile, with extreme drought and temperatures, and barren
rocky substrate; hence, the association with terrestrial fungi allowed
the algae ancestors of plants to successfully colonize the land (Selosse
et al. 2015). This initial symbiotic association was the prelude of
modern mycorrhizas (Feijen et al. 2018), the association between fungi
and root plants in which plants transfer carbon to fungi and receive
nutrients in turn (Smith and Read 2008). Today, this symbiosis is
present in 86\% of land plants species (Heijden et al. 2015), and based
on their structure and function can be classified in four major types:
arbuscular mycorrhiza (AM), ectomycorrhiza (EM), orchid mycorrhizal (OM)
and ericoid mycorrhizal (ER) (Brundrett 2002).

Ancestral state reconstruction and the fossil record suggest that the
ancestor of seed plants probably had AM associations (Redecker et al.
2000; Maherali et al. 2016). This is the most frequent mycorrhizal type
in plants (74\% of extant plant species) and is characterized by an
association with Glomeromycete fungi (Heijden et al. 2015). Between 100
and 200 million years ago, some lineages switched fungal partners to
several lineages of Basidiomycetes, forming what is described as the EM
associations (Brundrett 2002). The acquisition of EM resulted in new
root functional capabilities as freezing tolerance (Lehto et al. 2008),
which seem related to the dominance of EM angiosperms and gymnosperm in
cool forests (Brundrett 2002). Similarly, Orchidaceae and species within
the Ericaceae family recruited new fungal lineages and formed OM and ER
associations respectively. Orchids associate with fungal families
Ceratobasidiaceae, Tulasnellaceae and Sebacinaceae, which in addition to
nutrient exchange, promote seed germination which cannot germinate
without mycorrhizal support (Rasmussen 2002). Ericoid mycorrhizal
associations (ER), on the other hand, involve mainly fungi from
Sebacinales and Helotiales and are mostly frequent under acidic and
infertile heathland conditions (Perotto et al. 2002; Heijden et al.
2015). Finally, some lineages have lost their mycorrhizal associations
and became non-mycorrhizal (NM). This transition has frequently occurred
through an intermediate state of facultative arbuscular mycorrhiza (AM)
plants (Maherali et al. 2016). Some of NM lineages evolved alternative
resource-acquisition strategies (Werner et al. 2018) like cluster-roots
in Proteaceae (Neumann and Martinoia 2002) or parasitism in Loranthaceae
(Wilson and Calvin 2006).

Therefore, since the AM ancestral state some plant lineages have
followed different mycorrhizal evolutionary pathways: switching partner
(EM, OM and ER) or losing the association (Werner et al. 2018).
Evolutionary transitions to a novel mycorrhizal state might allow plant
lineages to access unexplored ecological resources, facilitating them to
colonize environments that were not available before, and possibly
enhancing their diversification rates. However, there are lineages in
which some species acquire a new mycorrhizal state and at the same time,
other species retain the ancestral state (AM) (Brundrett n.d.)
increasing the variability of mycorrhizal states, which might in fact
promote diversification of these lineages. Both hypotheses have not been
evaluated in plants; however, the few studies available from the fungal
perspective suggest that shifts in mycorrhizal associations might affect
diversification of involved partners (Sánchez-García and Matheny 2017;
Sato et al. 2017).

Even though mycorrhizal symbiosis has been pointed out as a key factor
in the evolution and diversification of land plants (Brundrett and
Tedersoo 2018; Feijen et al. 2018) this has not been evaluated before.
In this study we address the following questions: (1) Do the lineages
that established derived mycorrhizal associations present higher
diversification rates than the ones that retain the ancestral
mycorrhizal state? This investigates the idea of a key innovation
mechanism of diversification; (2) Is there a relationship between
mycorrhizal variability and diversification rates among different plant
lineages? This would investigate the idea that evolutionary lability
might increase diversification dynamics. To answer these questions, we
explored the relationship between the mycorrhizal state and the
diversification rates of several seed plant families.

\hypertarget{materials-and-methods}{%
\section{Materials and Methods}\label{materials-and-methods}}

\hypertarget{mycorrhizal-state-database}{%
\subsection{Mycorrhizal state
database}\label{mycorrhizal-state-database}}

To obtain information of plant species and their mycorrhizal state, we
used the FungalRoot database, a recently published global databank of
plant mycorrhizal associations (Soudzilovskaia et al. 2019). This
database compiles previous lists and surveys of plant species and their
mycorrhizal associations, including 36,303 records for 14,870 plant
species. Additionally, based on these empirical records and on expert
opinion, the authors proposed a list of mycorrhizal status at the plant
genus level, which contains 14,541 total genera, from which 12,558
correspond to seed plant genera that together results in information for
295,221 seed plant species.

Recently, Brundrett \& Tedersoo (2019) pointed out potential mistakes in
mycorrhizal type identification on large databases, and how these
misdiagnoses might lead to wrong conclusions. Although their approach
used to determine these errors (taxonomic approach; Brundrett (2017)) is
controversial (Guillermo Bueno et al. 2019; Sun et al. 2019) and the
proportion of errors they detected in databases is relatively low
(Brundrett and Tedersoo 2019), Brundrett and Tedersoo (2019) are right
to point out that caution must be taken when analyzing large databases
of plant mycorrhizal status. Therefore, in addition to the main analyses
that were conducted using the genus-level list, we evaluated if the
results were maintained when using only empirical data, by conducting
the same analyses using the species-level list. Additionally, in the
species-level list, the authors included remarks for 3,954 plant records
(out of 36,303), indicating potential mistakes or misidentification of
mycorrhizal associations in the original publication (see details in
Table Media 3 in Soudzilovskaia et al. (2019)). Then, to test the effect
of potential errors in the database, we conducted the analyses (i)
excluding and (ii) without excluding plant records that had remarks (see
Appendix S1 in Supporting Information). Furthermore, to assess the
effect of possible undetected errors in the genus-level dataset, we
introduced errors to the mycorrhizal state of 20\% of plant species (one
order of magnitude higher than the error estimated from Brundrett and
Tedersoo (2019)). The results obtained with the error-introduced
databases were similar than those derived from original data (Appendix
S2).

\hypertarget{family-mycorrhizal-state-and-diversity}{%
\subsection{Family mycorrhizal state and
diversity}\label{family-mycorrhizal-state-and-diversity}}

The genus-level list from FungalRoot database includes information for
genera belonging to 392 seed plant families. Before using this list, we
prepared the data as follows (i) Typo correction: removed entries with
spaces at the end, with double spaces or line breaks, (ii) matched
genera to families using the table \emph{Spermatophyta\_Genera.csv},
obtained from Zanne et al.~(2014), (iii) Used package ``taxize''
(Chamberlain et al. 2019) for R (R Core Team 2019) to fill in for genera
without family data and (iv) removed Ferns and Mosses. In the
genus-level list, genera were classified as AM, EM, NM, OM, ER, or with
multiple mycorrhizal status (i.e.~AM-EM, AM-NM). The genera that were
classified with multiple mycorrhizal status were classified as MIX, to
indicate that these genera presented more than one mycorrhizal state.
Instead of using MIX as a separate mycorrhizal status, we divided the
species richness of those genera equally into the types that composed
the MIX category, e.g.~if one genus had 100 species and was classified
originally as AM-NM, we would then add 50 species to the AM category and
50 species to the NM category prior to grouping all data per family. We
obtained the richness of each genus from The Plant List
({[}theplantlist.org{]}), and then calculated the number of species with
each mycorrhizal state within each family, discarding those genera that
had unknown mycorrhizal type. Each family was assigned a unique
mycorrhizal state (AM, EM, NM, ER or OM) when more than 60\% of species
sampled belonged to this mycorrhizal state. If no single state were
present in more than 60\% of species, the family was assigned as
``MIX'', to indicate no dominance of any mycorrhizal association. Other
thresholds for the assignment of family mycorrhizal state were tested
and the pattern was similar (50\%, 80\% and 100\%, Appendix S3).
However, we excluded from the analyses the families from which all
species belonged to genera that were classified as MIX (only 18
families) given that in this case there is no direct information for any
species about its specific mycorrhizal state (MS), and using those could
strongly bias our results given our methods. Given our methodological
choice to assign equal proportions of MS for those genera classified as
MIX, that would, by definition, fix the absolute value of diversity
index for all those families, and hence possibly remove any signal for a
potential association between the diversity index and diversification
rates (see below). Given that those genera only make up for the entirety
of very few families (only 18 out of 392), we decided to simply remove
those families from the main analysis (analyses without removing those
families presented similar results and are shown in Appendix S3). To
investigate the effect of mycorrhizal diversity in the diversification
dynamics we estimated the ``Mycorrhizal State Diversity Index'', which
is calculated by estimating the heterogeneity of the mycorrhizal states
in each family using the shannon diversity index.

\hypertarget{diversification-rates}{%
\subsection{Diversification rates}\label{diversification-rates}}

First, to explore the underlying diversification model behind plant seed
diversification, we assessed the association between age and richness
among seed plant families. Thus, following Sanchez-Reyes et al.~(2017),
we evaluated the association between stem age and richness, including
all seed plant families available (i.e.~without removing families
lacking information on mycorrhizal states) and correcting for
phylogenetic structure and not. Stem group ages of the families were
obtained from the dated molecular phylogeny of seed plants of Zanne et
al.~(2014) and the number of species of each family was obtained from
The Plant List ({[}\url{http://www.theplantlist.org}{]}). No association
was found between stem group age and richness, either considering or not
phylogenetic structure (R\textsuperscript{2} = 0.009;
R\textsuperscript{2} = 0.007, respectively; Fig.1), suggesting that
diversification rates significantly vary among clades (Sánchez-Reyes et
al. 2017) and justifying further investigation. Diversification rates
for each seed plant family were estimated using the method-of-moments
from Magallón \& Sanderson (2001) and stem group ages. Because the
relative contribution of extinction is unknown, we used two distinct
scenarios to characterize the relative extinction rates (\(\epsilon\)),
one with no extinction, \(\epsilon\) = 0.0 and another with high
extinction, \(\epsilon\) = 0.9. We are aware of more sophisticated and
direct methods (e.g.~BAMM; Rabosky, (2014)) to investigate the
association between trait states and diversification dynamics, but the
plant phylogeny is massively under-sampled at the species level.
Therefore, we decided to use simpler and less data hungry methods, and
to discuss our results in the light of the methods limitations.

The tree used in this study (Zanne et al.~2014) was build using a
Maximum Likelihood framework, and therefore consists in a single
topology, not allowing phylogenetic uncertainty to be readily
incorporated. That said, given the fact that the possible variations in
age estimates should, at this temporal scale, not really influence the
diversification estimates (rates are calculated using age in a log
scale), we believe this has no impact in our results. More importantly,
we replicated the same procedure using a different family-level
phylogeny (Harris and Davies 2016), allowing some level of phylogenetic
uncertainty to be addressed. This study estimates diversification rates
using the same source of information for species richness per family
(theplantlist.org), but with different values for family ages, therefore
rendering these rates slightly different from our estimates. As
expected, the results are virtually identical (figures S10, S11 and S12,
and table S23) and we show here only the results from the Zanne et
al.~(2014) study (all the results for the other phylogeny can be seen in
the supplemental material).

\hypertarget{phylogenetic-signal}{%
\subsection{Phylogenetic signal}\label{phylogenetic-signal}}

The seed plant phylogeny (Zanne et al. 2014) was pruned to obtain a
family level phylogeny, with one species per family as tips. From this
pruned phylogeny we calculated the phylogenetic signal of mycorrhizal
traits and diversification rates. For the continuous variables -
mycorrhizal diversity index and diversification rates - we calculated
phylogenetic signal using Pagel's Lambda (Pagel 1999) using the function
phylosig in the package phytools in R (Revell 2012). For the categorical
variable, mycorrhizal state, we estimated the phylogenetic signal using
the D parameter (Fritz and Purvis 2010) with the function phylo.d in
\emph{caper} package in R (Orme et al. 2018).

\hypertarget{statistical-analysis}{%
\subsection{Statistical analysis}\label{statistical-analysis}}

As some (but not all) of the mycorrhizal traits and diversification
rates showed significant phylogenetic signal (Appendix S4), we evaluated
the effect of mycorrhizal associations on diversification rates by both
considering and not the phylogenetic structure in the residuals. We
tested for potential differences in diversification rates between plant
families with different mycorrhizal states using both ANOVA and a
phylogenetic ANOVA using the function phylanova from phytools in R. Each
mycorrhizal state was used as group and their diversification rates as
response variable. Because the mycorrhizal states OR and ER only had one
and two family respectively, those were removed from this analysis.

To test for the relationship between mycorrhizal heterogeneity and
diversification rates we performed a linear model with raw data, and a
PGLS regression in the R package \emph{caper}) (Orme et al. 2018) with
diversification rates as response variable and mycorrhizal heterogeneity
as explanatory variable. For PGLS models we used the lambda value
obtained from the previous phylogenetic signal analysis. To further
explore the potential confounding effect and the association between
mycorrhizal association and diversification dynamics, we performed PGLS
regressions to assess the relationship between mycorrhizal diversity
index, age and species richness.

\hypertarget{results}{%
\section{Results}\label{results}}

The genus-level list from FungalRoot database contained information
about mycorrhizal state of 295,221 species that belong to 392 families
of seed plants. From these, the families OM (Orchidaceae) and ER
(Ericaceae and Diapensiaceae) were excluded due to lack of replication,
and 18 MIX families were excluded because of the inability to stablish
the proportion of mycorrhizal status within them (see Materials and
Methods). Then, we kept 372 families for the analyses. According to our
classification, using 60\% threshold for mycorrhizal state assignment,
290 families were AM (for example, Amaryllidaceae, Asteraceae and
Lamiaceae), 9 were EM (like Fagaceae, Nothofagaceae, Betulaceae and
Pinaceae), 46 were NM (such as Brassicaceae, Caryophyllaceae and
Juncaginaceae) and 27 were mixed (Fig. 2). Mixed families contain
species that retained the ancestral state (AM) and species that present
a different mycorrhizal state (EM or NM). There were three different
types of mixed families: 21 mixed families had AM and NM species (such
as Amaranthaceae, Cyperaceae and Juncaceae), four had AM and EM species
(Casuarinaceae, Hydrocharitaceae and Juglandaceae) and three had AM, EM
and NM (Goodeniaceae, Nyctaginaceae and Polygonaceae). The phylogenetic
signal strength differs among mycorrhizal states, but all mycorrhizal
states are phylogenetically clustered to some extent (Table S21).
Likewise, the phylogenetic signal of diversification rates was
significantly different from a random structure in rɛ = 0.0 and rɛ = 0.9
(Table S22). There was a significant difference in diversification rates
between the mycorrhizal states, irrespective of the extinction scenario
(standard ANOVA: r\textsubscript{$\epsilon$ = 0.0}: F = 7.25, p = 0.013;
r\textsubscript{$\epsilon$ = 0.9}: F = 7.35, p = 0.007; Fig. 3a and 3b),
which was observed in the ANOVA and in the phylogenetic ANOVA (Tables
S12 and S13). The a posteriori analysis of the ANOVA showed that
diversification of MIX families was significantly higher than that of AM
and NM families (Table S15) and the same tendency is observed when
correcting for the phylogenetic structure (Table S14). The ANOVA also
showed there was no significant difference in diversification rates
between the different types of mixed families
(r\textsubscript{$\epsilon$ = 0.0}: F = 0.67, p = 0.51;
r\textsubscript{$\epsilon$ = 0.9}: F = 0.97, p = 0.39).

The higher values of mycorrhizal diversity index were found in
Nyctaginaceae (1.09), Polygonaceae (0.98) and Rhizophoraceae (0.726),
while the lowest value was zero and it was observed in 275 families that
have all species in the same mycorrhizal state, like in Pinaceae (EM, n
= 255), Araucariaceae (AM, n = 38) and Droseraceae (NM, n = 189). There
was a positive correlation between mycorrhizal diversity index and
diversification rates, observed with the linear models and with the
PGLS, and under the two scenarios of extinction (Figure 4a and 4c). The
R\textsuperscript{2} are surprisingly high, and together with the
p-values of the models, are shown in each panel of Fig. 4. Mycorrhizal
diversity index had no correlation with age and a significant but very
low correlation with species richness (R\textsuperscript{2} = 0.002 and
0.01, respectively; Fig. 4b and 4d).

The additional analyses of adding a mycorrhizal misidentification to
20\% of the species, supported our main conclusions, which are the
positive association between mycorrhizal diversity index and
diversification rates, and MIX families having higher diversification
rates (Appendix S2). The additional analyses at the species level also
showed a positive association between mycorrhizal diversity index and
diversification rates (Tables S5 and S10). Although the ANOVA analysis
at the species level showed a similar tendency for most comparisons, it
did not show significant differences on diversification rates among
different mycorrhizal states for all thresholds (Fig. S1 and S3, and
tables S1 and S2).

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

The association with mycorrhizal fungi has been indicated as a key
acquisition in the evolution of plants, nevertheless its effect on
plants diversification has not been evaluated before. Here we presented
the first attempt to assess the relationship between mycorrhizal
associations and diversification rates of plants. Due to the
under-sampling of seed plants phylogeny and mycorrhizal state database,
we used a simple and conservative approach that allows us to tackle this
question.

Our results showed that there was no difference on diversification rates
between AM, EM and NM families (Fig. 3; Table S14 and S15). This shows
that families that acquired novel mycorrhizal associations (EM and NM)
do not have higher diversification rates than families that retained the
ancestral state (AM), contrary to what was expected in a scenario of key
innovation in mycorrhizal associations as a mechanism of
diversification. Thus, regarding our first question, the lineages that
established derived mycorrhizal associations do not differ in their
diversification rates from AM families. Contrary, our analyses showed
that families with mixed mycorrhizal state have higher diversification
rates than AM and NM families (Fig. 3, Table S14). Mixed families
included three subtypes of mixed: families with AM and NM species,
families with AM and EM species and families with AM, EM and NM species;
the three subtypes had higher diversification rates and there was no
significant difference on rates between them. This shows that regardless
of the mycorrhizal states that composed the mixed families, they have
the highest diversification rates, suggesting that it is the diversity
of mycorrhizal states that promotes diversification rather than a
specific mycorrhizal state.

In addition, there was a positive and significant association between
mycorrhizal diversity index and diversification rates, which does not
depend on our categorical criteria of mycorrhizal state assignment to
families. These associations with diversification rates, are observed
when correcting or not for the phylogenetic structure, suggesting that
the relationship is not due to phylogenetic relatedness between
families. Also, the patterns are observed under different scenarios of
extinction, and even with \(\epsilon\) = 0.9, where extinction could
have an important role, the relationship is conserved. Given that
diversification rates are determined by age and richness of the family,
the effect of those variables could have driven the relationship between
mycorrhizal heterogeneity and diversification rates. We observed no
significant correlation between mycorrhizal heterogeneity and age; and
we see a similar pattern with species richness, although the correlation
is significant, but the R\textsuperscript{2} is quite low (Fig. 4b, 4d).
This supports that mycorrhizal heterogeneity is mainly associated with
diversification rates, not with age or richness per se. In addition,
these patterns are also observed when we analyzed the species-level data
(Appendix S1).

Both results, the ANOVA for family mycorrhizal type and association
between mycorrhizal heterogeneity and diversification, suggest that
independent of which mycorrhizal state is involved, a higher
heterogeneity of mycorrhizal states in a family might promote
diversification rates. We interpret mycorrhizal heterogeneity as a
result from a higher evolutionary lability of the mycorrhizal states
within these families, which has been suggested to promote
diversification in other biotic interactions (Hardy and Otto 2014). Each
mycorrhizal state provides advantages to plants in certain environments
but not in others (Brundrett 2002), thus families that are composed by
species with different mycorrhizal states might have been able to switch
states in evolutionary time, making them able to evolve a higher
diversity of niches which would result in a higher diversification rate.
Under this scenario, mycorrhizal diverse families would have had more
chances to take advantage of a new ecological opportunity, than families
with most species within a single mycorrhizal state. It is interesting
to note that mycorrhizal diverse families have not only higher
diversification when compared to low diverse families with the ancestral
state, but also higher rates than families that have switched from the
ancestral state to one novel mycorrhizal state (NM and EM families).

The mycorrhizal diversity index might not capture well the effects of
mycorrhizal shifts on diversification rates if shifts occurred only once
within each family. However, we observed that mycorrhizal shifts in mix
families occurred multiple times, because more than 97\% of mixed
families contain genera that have multiple mycorrhizal states, this
means that the MS do not form monophyletic sub-clades and shifts occur
even below the genus level. This suggests that diversification rates are
not the result of a single mycorrhizal shift, but a result of high
lability of the mycorrhizal states within the mixed families. These
results together suggest that rather than a key innovation scenario, it
is the evolutionary variability of mycorrhizal state what promotes
diversification rates of plant seed families. Our results also highlight
the evolutionary role of specialization at different organization
levels: even if species are mycorrhizal specialized within a mixed
family, the possibility to switch to different mycorrhizal states might
increase the diversification of the family.

Because biodiversity dynamics could be rather complex, with clades
either expanding, at equilibrium and even declining in diversity, simple
metrics like the average rate of diversification might not be able to
separate them (Quental and Marshall 2010). The use of an average rate as
a descriptor of a clade diversification dynamics assumes (or at least
equates to) a scenario of expanding diversity (Quental and Marshall
2010), and it might be especially problematic if lineages have a
carrying capacity because the average rate might be diluted as time goes
by (Rabosky 2009). Moreover, with an average rate is not possible to
distinguish between speciation and extinction rates or to test directly
the effect of one trait on diversification dynamics. Ideally one would
use more complex tests, but that would require a lot more phylogenetic
data than what is currently available. Overall, we used a relatively
simple and limited macroevolutionary method and our conclusions arose
from a limited ecological and phylogenetic data. These conclusions might
be revisited in future studies, when more data on mycorrhizal states and
more complete phylogenies of plants are available.

Acknowledging the limitations of our study, the results suggest that a
higher diversity of mycorrhizal strategies promotes diversification of
lineages, possibly related with new ecological opportunities that each
mycorrhizal state provides to plants. Our results finally suggest that
the associations between soil fungi and plants has been key for plant
diversification, not only due to the foundational association that
allows plants colonize land (Pirozynski and Malloch 1975) but also for
further diversification of seed plant lineages.

\hypertarget{references}{%
\section{References}\label{references}}

\hypertarget{refs}{}
\begin{cslreferences}
\leavevmode\hypertarget{ref-brundrett_website_2008}{}%
Brundrett, M. n.d. Mycorrhizal associations: The web resource.
MYCORRHIZAL A: The Web Resource. WEBSITE.

\leavevmode\hypertarget{ref-brundrett_2002}{}%
Brundrett, M. C. 2002. Coevolution of roots and mycorrhizas of land
plants. The New Phytologist 154:275--304.

\leavevmode\hypertarget{ref-brundrett_2017}{}%
---------. 2017. Global diversity and importance of mycorrhizal and
nonmycorrhizal plants. Pages 533--556 \emph{in} L. Tedersoo, ed.
Biogeography of mycorrhizal symbiosis, Ecological studies (Vol. 230).
Springer International Publishing, Cham.

\leavevmode\hypertarget{ref-brundrett_2018}{}%
Brundrett, M. C., and L. Tedersoo. 2018. Evolutionary history of
mycorrhizal symbioses and global host plant diversity. The New
Phytologist 220:1108--1115.

\leavevmode\hypertarget{ref-brundrett_2019}{}%
Brundrett, M., and L. Tedersoo. 2019. Misdiagnosis of mycorrhizas and
inappropriate recycling of data can lead to false conclusions. The New
Phytologist 221:18--24.

\leavevmode\hypertarget{ref-chamberlain_2019}{}%
Chamberlain, S., E. Szoecs, Z. Foster, Z. Arendsee, C. Boettiger, K.
Ram, I. Bartomeus, et al. 2019. Taxize: Taxonomic information from
around the web.

\leavevmode\hypertarget{ref-eriksson_1992}{}%
Eriksson, O., and B. Bremer. 1992. Pollination systems, dispersal modes,
life forms, and diversification rates in angiosperm families. Evolution
46:258--266.

\leavevmode\hypertarget{ref-feijen_2018}{}%
Feijen, F. A. A., R. A. Vos, J. Nuytinck, and V. S. F. T. Merckx. 2018.
Evolutionary dynamics of mycorrhizal symbiosis in land plant
diversification. Scientific Reports 8:10698.

\leavevmode\hypertarget{ref-fritz_2010}{}%
Fritz, S. A., and A. Purvis. 2010. Selectivity in mammalian extinction
risk and threat types: A new measure of phylogenetic signal strength in
binary traits. Conservation Biology 24:1042--1051.

\leavevmode\hypertarget{ref-guillermobueno_2019}{}%
Guillermo Bueno, C., M. Gerz, M. Zobel, and M. Moora. 2019. Conceptual
differences lead to divergent trait estimates in empirical and taxonomic
approaches to plant mycorrhizal trait assignment. Mycorrhiza 29:1--11.

\leavevmode\hypertarget{ref-hardy_2014}{}%
Hardy, N. B., and S. P. Otto. 2014. Specialization and generalization in
the diversification of phytophagous insects: Tests of the musical chairs
and oscillation hypotheses. Proceedings. Biological Sciences / the Royal
Society 281.

\leavevmode\hypertarget{ref-Harris_Davies_2016}{}%
Harris, L. W., and T. J. Davies. 2016. A complete fossil-calibrated
phylogeny of seed plant families as a tool for comparative analyses:
Testing the ``time for speciation'' hypothesis. Plos One 11:e0162907.

\leavevmode\hypertarget{ref-vanderheijden_2015}{}%
Heijden, M. G. A. van der, F. M. Martin, M.-A. Selosse, and I. R.
Sanders. 2015. Mycorrhizal ecology and evolution: The past, the present,
and the future. The New Phytologist 205:1406--1423.

\leavevmode\hypertarget{ref-lehto_2008}{}%
Lehto, T., A. Brosinsky, H. Heinonen-Tanski, and T. Repo. 2008. Freezing
tolerance of ectomycorrhizal fungi in pure culture. Mycorrhiza
18:385--392.

\leavevmode\hypertarget{ref-magalln_2001}{}%
Magallón, S., and M. J. Sanderson. 2001. Absolute diversification rates
in angiosperm clades. Evolution 55:1762--1780.

\leavevmode\hypertarget{ref-maherali_2016}{}%
Maherali, H., B. Oberle, P. F. Stevens, W. K. Cornwell, and D. J.
McGlinn. 2016. Mutualism persistence and abandonment during the
evolution of the mycorrhizal symbiosis. The American Naturalist
188:E113--E125.

\leavevmode\hypertarget{ref-moore_2007}{}%
Moore, B. R., and M. J. Donoghue. 2007. Correlates of diversification in
the plant clade dipsacales: Geographic movement and evolutionary
innovations. The American Naturalist 170 Suppl 2:S28--55.

\leavevmode\hypertarget{ref-neumann_2002}{}%
Neumann, G., and E. Martinoia. 2002. Cluster roots--an underground
adaptation for survival in extreme environments. Trends in Plant Science
7:162--167.

\leavevmode\hypertarget{ref-omeara_2016}{}%
O'Meara, B. C., S. D. Smith, W. S. Armbruster, L. D. Harder, C. R.
Hardy, L. C. Hileman, L. Hufford, et al. 2016. Non-equilibrium dynamics
and floral trait interactions shape extant angiosperm diversity.
Proceedings. Biological Sciences / the Royal Society 283.

\leavevmode\hypertarget{ref-orme_software_2018}{}%
Orme, D., R. Freckleton, G. Thomas, T. Petzoldt, S. Fritz, N. Isaac, and
W. Pearse. 2018. Caper: Comparative analyses of phylogenetics and
evolution in r. computer software.PROGRAM\_DOCUMENTATION, Online
reference manual.

\leavevmode\hypertarget{ref-pagel_1999}{}%
Pagel, M. 1999. Inferring the historical patterns of biological
evolution. Nature 401:877--884.

\leavevmode\hypertarget{ref-perotto_2002}{}%
Perotto, S., M. Girlanda, and E. Martino. 2002. Ericoid mycorrhizal
fungi: Some new perspectives on old acquaintances. Pages 41--53
\emph{in} S. E. Smith and F. A. Smith, eds. Diversity and integration in
mycorrhizas. Springer Netherlands, Dordrecht.

\leavevmode\hypertarget{ref-pirozynski_1975}{}%
Pirozynski, K. A., and D. W. Malloch. 1975. The origin of land plants: A
matter of mycotrophism. Bio Systems 6:153--164.

\leavevmode\hypertarget{ref-quental_2010}{}%
Quental, T. B., and C. R. Marshall. 2010. Diversity dynamics: Molecular
phylogenies need the fossil record. Trends in Ecology \& Evolution
25:434--441.

\leavevmode\hypertarget{ref-rabosky_2009}{}%
Rabosky, D. L. 2009. Ecological limits and diversification rate:
Alternative paradigms to explain the variation in species richness among
clades and regions. Ecology Letters 12:735--743.

\leavevmode\hypertarget{ref-rabosky_2014}{}%
---------. 2014. Automatic detection of key innovations, rate shifts,
and diversity-dependence on phylogenetic trees. Plos One 9:e89543.

\leavevmode\hypertarget{ref-rasmussen_2002}{}%
Rasmussen, H. N. 2002. Recent developments in the study of orchid
mycorrhiza. Pages 149--163 \emph{in} S. E. Smith and F. A. Smith, eds.
Diversity and integration in mycorrhizas. Springer Netherlands,
Dordrecht.

\leavevmode\hypertarget{ref-rcoreteam_software_2019}{}%
R Core Team. 2019. R: A language and environment for statistical
computing. computer software.PROGRAM\_DOCUMENTATION, R Foundation for
Statistical Computing, Vienna, Austria.

\leavevmode\hypertarget{ref-redecker_2000}{}%
Redecker, D., R. Kodner, and L. E. Graham. 2000. Glomalean fungi from
the ordovician. Science 289:1920--1921.

\leavevmode\hypertarget{ref-revell_2012}{}%
Revell, L. J. 2012. Phytools: An r package for phylogenetic comparative
biology (and other things). Methods in Ecology and Evolution 3:217--223.

\leavevmode\hypertarget{ref-snchezgarca_2017}{}%
Sánchez-García, M., and P. B. Matheny. 2017. Is the switch to an
ectomycorrhizal state an evolutionary key innovation in mushroom-forming
fungi? A case study in the tricholomatineae (agaricales). Evolution
71:51--65.

\leavevmode\hypertarget{ref-snchezreyes_2017}{}%
Sánchez-Reyes, L. L., H. Morlon, and S. Magallón. 2017. Uncovering
higher-taxon diversification dynamics from clade age and
species-richness data. Systematic Biology 66:367--378.

\leavevmode\hypertarget{ref-sato_2017}{}%
Sato, H., A. S. Tanabe, and H. Toju. 2017. Host shifts enhance
diversification of ectomycorrhizal fungi: Diversification rate analysis
of the ectomycorrhizal fungal genera strobilomyces and afroboletus with
an 80-gene phylogeny. The New Phytologist 214:443--454.

\leavevmode\hypertarget{ref-selosse_1998}{}%
Selosse, M. A., and F. Le Tacon. 1998. The land flora: A
phototroph-fungus partnership? Trends in Ecology \& Evolution 13:15--20.

\leavevmode\hypertarget{ref-selosse_2015}{}%
Selosse, M.-A., C. Strullu-Derrien, F. M. Martin, S. Kamoun, and P.
Kenrick. 2015. Plants, fungi and oomycetes: A 400-million year affair
that shapes the biosphere. The New Phytologist 206:501--506.

\leavevmode\hypertarget{ref-smith_2008}{}%
Smith, S. E., and D. J. Read. 2008. Mycorrhizal symbiosis. Elsevier.

\leavevmode\hypertarget{ref-soudzilovskaia_2019}{}%
Soudzilovskaia, N. A., S. Vaessen, M. Barcelo, J. He, S. Rahimlou, K.
Abarenkov, M. C. Brundrett, et al. 2019. FungalRoot: Global online
database of plant mycorrhizal associations. bioRxiv.

\leavevmode\hypertarget{ref-strulluderrien_2018}{}%
Strullu-Derrien, C., M.-A. Selosse, P. Kenrick, and F. M. Martin. 2018.
The origin and evolution of mycorrhizal symbioses: From palaeomycology
to phylogenomics. The New Phytologist 220:1012--1030.

\leavevmode\hypertarget{ref-sun_2019}{}%
Sun, T., H. Zhang, and Z. Wang. 2019. Reply to tedersoo et al.: plant
species within the same family or genus can have different mycorrhizal
types? Proceedings of the National Academy of Sciences of the United
States of America 201903868.

\leavevmode\hypertarget{ref-vamosi_2018}{}%
Vamosi, J. C., S. Magallón, I. Mayrose, S. P. Otto, and H. Sauquet.
2018. Macroevolutionary patterns of flowering plant speciation and
extinction. Annual review of plant biology 69:685--706.

\leavevmode\hypertarget{ref-werner_2018}{}%
Werner, G. D. A., J. H. C. Cornelissen, W. K. Cornwell, N. A.
Soudzilovskaia, J. Kattge, S. A. West, and E. T. Kiers. 2018. Symbiont
switching and alternative resource acquisition strategies drive
mutualism breakdown. Proceedings of the National Academy of Sciences of
the United States of America 115:5229--5234.

\leavevmode\hypertarget{ref-willis_2014}{}%
Willis, C. G., C. C. Baskin, J. M. Baskin, J. R. Auld, D. L. Venable, J.
Cavender-Bares, K. Donohue, et al. 2014. The evolution of seed dormancy:
Environmental cues, evolutionary hubs, and diversification of the seed
plants. The New Phytologist 203:300--309.

\leavevmode\hypertarget{ref-wilson_2006}{}%
Wilson, C. A., and C. L. Calvin. 2006. An origin of aerial branch
parasitism in the mistletoe family, loranthaceae. American Journal of
Botany 93:787--796.

\leavevmode\hypertarget{ref-zanne_2014}{}%
Zanne, A. E., D. C. Tank, W. K. Cornwell, J. M. Eastman, S. A. Smith, R.
G. FitzJohn, D. J. McGlinn, et al. 2014. Three keys to the radiation of
angiosperms into freezing environments. Nature 506:89--92.
\end{cslreferences}

\newpage

\hypertarget{figure-captions}{%
\section{Figure captions}\label{figure-captions}}

Figure 1: Relationship between number of species and age for each
lineage when compared to the confidence intervals based on the global
diversification rate of seed plants. The solid and dashed lines
represent the expected richness for epsilon = 0 and epsilon = 0.9,
respectively. The color of the points represents the mycorrhizal state
for the 60\% threshold.

Figure 2: Family-level, time-calibrated phylogeny for the 367 seed plant
families included in the analyses. For each family, the proportion of
species within each mycorrhizal state is represented in the rose-to-red
boxes, AM: Arbuscular mycorrhiza, EM: Ectomycorrhiza and NM:
non-mycorrhizal. The mycorrhizal state diversity index (MSDI) is
represented in the purple boxes and the diversification rates (r) are
shown in the green boxes. To illustrate the timescale of the phylogeny,
the width of concentric white and gray circles represents 50 million
years.

Figure 3: Relationship between mycorrhizal state and diversification
rates. a) diversification rate estimated with \(\epsilon\) (relative
extinction fraction) = 0 and b) diversification rate estimated with
\(\epsilon\)= 0.9. AM: Arbuscular mycorrhiza, EM: Ectomycorrhiza, NM:
non-mycorrhizal and MIX (families with no dominance of any specific
mycorrhizal association). The size of the points indicates the
Mycorrhizal State Diversity Index value for each lineage, indicating a
predominance of larger indices with higher diversification rates.

Figure 4: Scatterplots showing the relationship between mycorrhizal
diversity index and diversification rates (a and c), species richness
(b) and age family (d). Diversification rates were estimated with
\(\epsilon\) (relative extinction fraction) = 0 (a) and with
\(\epsilon\) = 0.9 (c). The red and blue lines indicate the results of a
linear model and a phylogenetic generalized least squares (PGLS) fit,
respectively. Respective p-values and R\textsuperscript{2} are shown in
each panel.

\newpage

\hypertarget{figures}{%
\section{Figures}\label{figures}}

\includegraphics{../output/figs/magsand_stem_nolabel.pdf}

\newpage

\includegraphics{../output/tree_fig/phylo_genus_final_resized.pdf}

\newpage

\includegraphics{../output/figs/boxplots_netdiv_myctype_r09.pdf}

\newpage

\includegraphics{../output/figs/scatterplots_lm_pgls_stem_r09.pdf}

\end{document}
